### PR TITLE
Integrate with march iv

### PR DESCRIPTION
**Pull request by JitskedeVries**
This new feature loads the urdf and visualizes the CoM of the whole robot.

Must be run through:
roslaunch calculation-CoM cog.launch This launches the urdf, placed in the march_description package. It launches the robot state publisher and the joint state publisher, which also opens a GUI.
rosrun calculation-CoM com.py This runs the node, which gets all the links from the urdf and calculates the CoM which is published in an message.
rviz This opens rviz. If you load the robotmodel and the point which is calculated by the com.py node, you can see everything.

### CODE REVIEW

File: cog.launch [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <!-- Launch robot_state_publisher -->
+    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" args="" output="screen" respawn="true">
+    </node>
+
+    <!-- Launch robot_state_publisher -->
+    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" args="" output="screen" respawn="true">
+        <param name="use_gui" value="true"/>
+    </node>
+
+</launch>
```
Comment (Ishadijcks) [inline cog.launch]: Will this be the final launchfile or just a demo?

If its final, please remove everything not related to your node.
If a demo, please name it accordingly

Comment (JitskedeVries) [reply cog.launch]: In what file should I add the word demo?

Comment (Ishadijcks) [reply cog.launch]: The file I'm commenting on

File: cog-test.launch [hunk 2]
```diff
@@ -9,6 +9,7 @@
     </node>
 
     <!-- Launch robot_state_publisher -->
+    <!-- The GUI can be removed if this feature is totally integrated with the exoskeleton. Since then exoskeleton joints states will be published. -->
```
Comment (Ishadijcks) [inline cog-test.launch]: Add a @TODO

File: src/center_of_mass/com.py [hunk 3]
```diff
@@ -50,7 +50,7 @@ def calculator(self):
         pub = rospy.Publisher('/com', Marker, queue_size=1)
         
         rate = rospy.Rate(10)
-        rospy.sleep()
+        rospy.sleep(10)
```
Comment (Ishadijcks) [inline src/center_of_mass/com.py]: I think you mean `rate.sleep()` to control the rate at which you publish?

Comment (JitskedeVries) [reply src/center_of_mass/com.py]: It gives an error specific on the absence of a number

Comment (Ishadijcks) [reply src/center_of_mass/com.py]: What kind of error?

Comment (Ishadijcks) [reply src/center_of_mass/com.py]: https://answers.ros.org/question/264812/explanation-of-rospyrate/

Comment (JitskedeVries) [reply src/center_of_mass/com.py]: TypeError: sleep() takes exactly 1 argument (0 given)

Comment (JitskedeVries) [reply src/center_of_mass/com.py]: Gefixt

**Pull request merged**